src/utils/generateUsageSnippets.js

```diff
@@ -33,7 +33,7 @@ resp = fs.get_offline_features(vector)`
       {
         title: 'Getting online features:',
         code: `svc = fs.get_online_feature_service("<vector-uri>")
-resp = svc.get([{"customer_id": "42"}, {"customer_id": "50"}])`
+resp = svc.get([{"${currentFeatureSet.spec.entities[0].name ?? ''}": <value>}])`
```

Comment:
In case the entities list is empty then `spec.entities[0]` is `undefined` and the above line will throw an error.
You can fix it like this:
```suggestion
resp = svc.get([{"${currentFeatureSet.spec.entities[0]?.name ?? ''}": <value>}])`
```